classes/note/Note.php

```diff
@@ -3,173 +3,145 @@
 /**
  * @file classes/note/Note.php
  *
- * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2000-2021 John Willinsky
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
```

Comment:
@asmecher Can you confirm the practice on this? I noticed in the `Notification.php` changes you removed the initial year.

Reply:
I don't think it matters at ton :) but I try to bump the end year to the current year when I modify something.
In `Notification.php`, I removed the start year and just left `2024` because it was a complete rewrite of the file to a new implementation. But I almost never do that.

Reply:
Thanks for clarifying - since this one is also a rewrite I think it makes sense to leave it as `2024`.